api/test/import-data/xlsx-uploads/sourcing-records-import.spec.ts

```diff
@@ -27,9 +27,10 @@ describe('Sourcing Records import', () => {
   let adminRegionRepository: AdminRegionRepository;
   let sourcingLocationRepository: SourcingLocationRepository;
   let sourcingRecordRepository: SourcingRecordRepository;
-  let sourcingRecordGroupRepository: SourcingLocationGroupRepository;
+  let sourcingLocationGroupRepository: SourcingLocationGroupRepository;
 
   beforeAll(async () => {
+    jest.setTimeout(10000);
```

Comment:
no, this is evil AF. set it in jest-config.json

Reply:
@tiagojsag IMO we need this timeOut increase just in these XLSX uploads tests so far, as these are 'heavy' ops, no need for this timeOut increase to be applied to the rest of the tests (Mostly simple CRUDs)

Applying them for all test cases could hide some sign of sickness in the flow

Reply:
fair point, but then you need to reset the previous value on after()